### PR TITLE
Task dl/j122 add default image component for user podcasts

### PR DESCRIPTION
**Pull request by Denidzio**
Screenshot from 2021-08-11 22-32-50
Screenshot from 2021-08-11 22-32-16
Screenshot from 2021-08-11 22-32-22
Screenshot from 2021-08-11 22-32-28

### CODE REVIEW

File: frontend/src/common/enums/ui/default-image-size.enum.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+enum DefaultImageSize {
```
Comment (ghost) [inline frontend/src/common/enums/ui/default-image-size.enum.ts]: we don't want it

File: frontend/src/components/common/default-image/common/constants/constants.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const SYMBOLS_COUNT_IN_LABEL = 1;
+const DEFAULT_LABEL = 'Undefined';
+
+const SMALL_DEFAULT_IMAGE: React.CSSProperties = {
```
Comment (ghost) [inline frontend/src/components/common/default-image/common/constants/constants.ts]: let's do it thought props for the component

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by Denidzio** [comment]
[Add default image component for user/podcasts](https://trello.com/c/ZiT1nEhT/122-add-default-image-component-for-user-podcasts)

**Comment by Denidzio** [comment]
Screenshot from 2021-08-12 13-35-18
Screenshot from 2021-08-12 13-35-30
Screenshot from 2021-08-12 13-36-23

**Review by artem-totality — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**